draft-ietf-ppm-dap.md

```diff
@@ -1291,44 +1295,70 @@ struct {
 } HpkeConfig;
 
 opaque HpkePublicKey<0..2^16-1>;
-uint16 HpkeAeadId; /* Defined in [HPKE] */
-uint16 HpkeKemId;  /* Defined in [HPKE] */
-uint16 HpkeKdfId;  /* Defined in [HPKE] */
+uint16 HpkeAeadId;
+uint16 HpkeKemId;
+uint16 HpkeKdfId;
 ~~~
 
-Aggregators MUST allocate distinct id values for each `HpkeConfig` in an
+The possible values for `HpkeAeadId`, `HpkeKemId` and `HpkeKdfId` are as defined
+in {{!HPKE, Section 7}}.
+
+Aggregators MUST allocate distinct `id` values for each `HpkeConfig` in an
 `HpkeConfigList`.
 
-The Client MUST abort if any of the following happen for any HPKE config
-request:
+The Client MUST abort if:
 
-* the GET request did not return a valid HPKE config list;
+* the response is not a valid HPKE config list;
 * the HPKE config list is empty; or
-* no HPKE config advertised by the Aggregator specifies a supported a KEM, KDF,
-  or AEAD algorithm triple.
-
-Aggregators SHOULD use HTTP caching to permit client-side caching of this
-resource {{!RFC5861}}. Aggregators SHOULD favor long cache lifetimes to avoid
-frequent cache revalidation, e.g., on the order of days. Aggregators can control
-this cached lifetime with the Cache-Control header, as in this example:
```

Comment:
I remixed the caching discussion: I think the appropriate reference is RFC 9111, no need to point to 5861. Further I moved the `Cache-Control` example into the example of an HPKE config request and response pair, below.